Edited Sandeep Panda transcript for content and formatting (#65)
mgreiler/se-unlocked: Transcripts/EP_04_Sandeep.md

# Transcript Episode Sandeep Panda

**Michaela:** [00:00:00] Hello, and welcome to the Software Engineering Unlocked 
podcast. I'm your host, Dr. McKayla and I'm opening the doors to the top companies 
and thought leaders around the world. Today, I speak with Sandeep Panda about 
starting a developer community from scratch, his entrepreneurial journey, 
remote work, and software development practices at his startup.

I'm super thrilled for this awesome third episode. I hope you enjoy it. If you 
like it, support my work by spreading the word. Tell your friends about the 

> podcast shar it on Twitter, LinkedIn, and Facebook. I promise I will do a happy 

Reviewer: ```suggestion
podcast share it on Twitter, LinkedIn, and Facebook. I promise I will do a happy 
```

dance each time you do it. 

Today, I have the pleasure to talk to Sandeep Panda, the founder and CEO of 
Hashnode, a friendly and inclusive developer community/ Hashnode is a startup 
that's fully remote just in February to shift it from a Q and A side to provide
plugin functionality for developers. So there's plenty to talk about here, but
yeah, let's start at the beginning. Sandeep, you're running a developer community.
Are you a developer yourself? 

**Sandeep:** [00:00:59] Thanks for having me McKayla. I'm honored to be here. 
So, uh, yeah, to answer your question, I am a developer and I love coding. 
And beside that, I'm the founder of the company called Hashnode and we are a 
team of six running, a friendly and inclusive community for engineers. 

**Michaela:** [00:01:16] So how did you become a developer? What was your way?

> Are yourself taught developer or did you go to University? 

Reviewer: ```suggestion
Are you a self-taught developer or did you go to university? 
```

**Sandeep:** [00:01:22] So I studied computer science for four years, uh, from 

> 2009 to 2013. And that kind of, uh, give me the post to become an engineer and 

Reviewer: ```suggestion
2009 to 2013. And that kind of, uh, give me the boost to become an engineer and 
```
